test/qc_model_tests.jl

```diff
@@ -121,6 +171,36 @@ end
 
 end
 
+@testset "Tests: 3-qubit CRX gate decomposition" begin
+
+    params = Dict{String, Any}(
+        "num_qubits" => 3, 
+        "depth" => 2,    
+        "elementary_gates" => ["CU3_12", "CU3_23", "CU3_13", "Identity"],  
+        "target_gate" => QCO.get_full_sized_gate("CRX_13", 3, target_gate = QCO.RXGate(pi/4)),
```

Comment:
Though I understand get_full_sized_gate is more of an internal function for QCOpt, I am just wondering if is there a neater way to automatically recognize that the target_gate is indeed an RXGate. As in, shouldn't it be sufficient to just enter "CRX_13", num_qubits and the discretization angle? Something to think before finalizing on this format.

Reply:
@lockwo Another issue is, if the we use this function in this wrong way, say "QCO.get_full_sized_gate("CRX_13", 3, target_gate = QCO.RYGate(pi/4))", function will still return some garbage which can lead to issues and may be difficult to recognize at a later stage?